README.md

```diff
@@ -23,6 +23,13 @@ _Note: Support for building the SDK with either Ant or Maven has been made. Plea
 * hamcrest-library-1.3.jar  : unit testing
 * jmock-2.6.0.jar           : unit testing
 
+### Migrating from older versions
+Since August 2018, the Authorize.Net API has been reorganized to be more merchant focused. AuthorizeNet AIM, ARB, CIM, Transaction Reporting and SIM classes have all been deprecated in favor of `net\authorize\api` . To see the full list of mapping of new features corresponding to the deprecated features, you can see [MIGRATING.md](MIGRATING.md).
+
+### Contribution
+ - If need information or clarification about any Authorize.Net features, please create an issue for it. Also you can search for it in the [Authorize.Net developer community](https://community.developer.authorize.net/).
```

Comment:
1- If you need
2- you can search in the (remove 'for it')

Reply:
@ashtru I have updated those changes. Thanks Ashu.